### PR TITLE
FLOW-13887 Add endpoint to synchronize Mailchimp contacts

### PR DESCRIPTION
**Pull request by l-pawliczak**

### CODE REVIEW

File: src/service/MailchimpSyncIntegrationService/MailchimpController.ts [hunk 1]
```diff
@@ -63,4 +63,11 @@ export class MailchimpController extends APIClient {
     async updateSettings(settings: MailchimpServiceTypes.Settings) {
         return this.invokeApiWithErrorHandling<MailchimpServiceTypes.Settings>(`/settings`, 'PUT', settings);
     }
+
+    /**
+     * Synchronize Mailchimp contacts
+     */
+    async synchronizeContacts() {
+        return this.invokeApiWithErrorHandling(`/publish`, "POST");
```
Comment (ncmbch) [inline src/service/MailchimpSyncIntegrationService/MailchimpController.ts:71]: What about the <type> ?

**Review by ncmbch — CHANGES_REQUESTED** [review]

File: src/service/MailchimpSyncIntegrationService/MailchimpController.ts (hunk 1 above)
Comment (ncmbch) [reply src/service/MailchimpSyncIntegrationService/MailchimpController.ts:71]: type*

Comment (ffjanhoeck) [reply src/service/MailchimpSyncIntegrationService/MailchimpController.ts:71]: And wrong

Comment (l-pawliczak) [reply src/service/MailchimpSyncIntegrationService/MailchimpController.ts]: Added void type and changed " to '

File: src/service/MailchimpSyncIntegrationService/MailchimpService.Types.ts [hunk 2]
```diff
@@ -13,6 +13,12 @@ export namespace MailchimpServiceTypes {
         listIds: string[];
     }
 
+    export interface SyncStatus {
+        companyId: string;
+        syncDirection: SyncDirection;
+        finished: boolean;
+    }
+
     export enum SyncDirection {
```
Comment (JakubJagosz) [inline src/service/MailchimpSyncIntegrationService/MailchimpService.Types.ts:22]: Please add type 'BOTH' here

**Review by ffjanhoeck — APPROVED** [review]

**Pull request merged**